docs_zh-CN/tutorials/customize_losses.md

```diff
@@ -1 +1,106 @@
 # 教程 6: 自定义损失函数
+
+MMDetection 为用户提供了不同的损失函数。但是默认的配置可能无法适应不同的数据和模型，所以用户可能会希望修改某一个损失函数来适应新的情况。
+
+本教程首先详细的解释计算损失的过程然后给出一些关于如何修改每一个步骤的指导。对损失的修改可以被分为微调和加权。
+
+
```

Comment:
delete this line

Reply:
I'm sorry that I make you confused. What I mean is you need to delete the Blank line, not the text